pkgs/applications/misc/sleepyhead/default.nix

```diff
@@ -1,9 +1,9 @@
-{ stdenv, fetchgit, qt5, zlib, libGLU, libX11 }:
+{ stdenv, fetchgit, qt5, zlib, libGLU, libX11, wrapQtAppsHook }:
```

Comment:
```suggestion
{ lib, mkDerivation, fetchgit, zlib, libGLU, libX11, qtbase, qtwebkit, qtserialport, wrapQtAppsHook }:
```

Reply:
Used and tested your suggestion on Darwin